llrt_modules/src/time.rs

```diff
@@ -0,0 +1,26 @@
+use std::{
+    sync::atomic::{AtomicU64, Ordering},
+    time::SystemTime,
+};
+
```

Comment:
Why can’t we keep usize?

Reply:
usize is a u32 on a 32bits system, while I dont claim support for those it still is not a good practice to convert between usize and u64. It wont make a difference on 64bits system.

Reply:
Sure. 32bit support is not prioritized anyway!